docs/src/pages/guides/typescript/typescript.md

```diff
@@ -179,7 +179,8 @@ interface Props {
 However this isn't very [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) because it requires you to maintain the class names (`'root'`, `'paper'`, `'button'`, ...) in two different places. We provide a type operator `WithStyles` to help with this, so that you can just write:
 
 ```ts
-import { WithStyles, createStyles } from '@material-ui/core';
+import { createStyles } from '@material-ui/styles';
```

Comment:
Let's remove the usage here as well.

Reply:
Looks like if we remove this we should update the whole page, as it actually talks about how to solve typescript issues for `withStyles`. Should we recommend casting to `const` there too?

Reply:
Yeah, let's do this another time.